hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptor.java

```diff
@@ -208,11 +219,11 @@ public void testSubmitApplication()
   }
 
   private SubmitApplicationRequest mockSubmitApplicationRequest(
-      ApplicationId appId) {
+      ApplicationId appId, int priority) {
```

Comment:
Can we add one with APP_PRIORITY_ZERO as default instead of having to modify every call?

Reply:
No problem, I will fix the code.